inst/extdata/installRDependencies.sh

```diff
@@ -1,26 +0,0 @@
-#!/bin/bash
-Rscript -e 'remove.packages("curl")' || true
```

Comment:
Jenkins doesn't have a way to run R directly? Seems like if it can run `bash`, it should be able to run `Rscript`. Doesn't matter, just surprised.